This patch allows the dbm module to be built using the compatibility mode of
gdbm. It will not be needed any more with Python 2.7.4.
--- setup.py	2013-04-06 00:53:37.000000000 +0200
+++ setup.py.new	2013-04-06 19:55:05.000000000 +0200
@@ -1158,10 +1158,14 @@
             for cand in dbm_order:
                 if cand == "ndbm":
                     if find_file("ndbm.h", inc_dirs, []) is not None:
-                        # Some systems have -lndbm, others don't
+                        # Some systems have -lndbm, some have -lgdbm_compat,
+                        # others have no particular linker flags.
                         if self.compiler.find_library_file(lib_dirs,
                                                                'ndbm'):
                             ndbm_libs = ['ndbm']
+                        elif self.compiler.find_library_file(lib_dirs,
+                                                             'gdbm_compat'):
+                            ndbm_libs = ['gdbm_compat']
                         else:
                             ndbm_libs = []
                         print "building dbm using ndbm"
